README.md

```diff
@@ -101,6 +101,8 @@ entities:
   baseClass: null
   # Protocols adopted by entities
   protocols: ["Codable"]
+  # Autogenerate Identifiable conformance if available
+  autogenerateIdentifiableConfomance: false
```

Comment:
Please can we rename this to `isGeneratingIdentifiableConformance`. 

I'm not personally a huge fan of using Swifty naming conventions in yaml but it keeps it consistent with everything else 🙏